src/bin/cli.rs

```diff
@@ -26,6 +26,10 @@ struct Cli {
 
 #[derive(Subcommand, Debug)]
 enum Command {
+    Ping {
+        /// Message to ping
+        message: Option<String>,
```

Comment:
Would it make sense to make this struct match the one in cmd and rename this field to `msg`?